.circleci/config.yml

```diff
@@ -164,9 +169,17 @@ jobs:
           command: |
             export IMAGE_TAG=$([[ "$CIRCLE_BRANCH" == "staging" ]] && echo "staging-candidate" || echo "discardable")
             export SNYK_MONITOR_IMAGE_TAG="${IMAGE_TAG}-${CIRCLE_SHA1}"
-            export SNYK_OPERATOR_VERSION="0.0.1"
+            export SNYK_OPERATOR_VERSION="0.0.1-${CIRCLE_SHA1}"
```

Comment:
this is what's using the ```upload_operator``` step?

Reply:
Not sure what is the question, the upload_operator step matches this versioning scheme!
The operator-sdk and Quay don't allow us to use just a tag, but something that resembles x.y.z-tag is OK